README.md

```diff
@@ -252,7 +259,7 @@ This stores a new Credential in the Credential Store and enables a silent access
 Upon return to the site to be logged into, the site runs the following:
 
 ```js
-let credentials = navigator.credentials.crossSiteRequests.getAllowed();
+let credentials = IdentityCredential.requests.getAllowed();
 for (let credential in credentials) {
   navigator.credentials.store(credential);
   await document.requestStorageAccess("cross-site" : credential);
```

Comment:
This doesn't seem needed if we just autogrant storage access without relying on the credential itself to do it.